ramls/types/resources/resourcePutDataAttributes.json

```diff
@@ -139,7 +139,7 @@
     "proxy": {
       "type": "object",
       "description": "Proxy. Note that this attribute can be updated both for CUSTOM AND MANAGED RESOURCES.",
-      "$ref": "../proxy.json"
+      "$ref": "../proxies/proxyUrl.json"
```

Comment:
Are you sure we need this on PUT?
I can't see this property in PUT schema on docs.